addon/service.js

```diff
@@ -25,8 +25,12 @@ export default Parent.extend(Ember.Evented, {
   t: function(key, data = {}) {
     const locale = this.get('_locale');
     Ember.assert("I18n: Cannot translate when locale is null", locale);
+    const count = get(data, 'count');
 
-    const template = locale.getCompiledTemplate(key, get(data, 'count'));
+    let defaults = makeArray(get(data, 'default'));
```

Comment:
This could be `const`, but `let` is just fine. If we ever come back to this function, we can change it then.